src/Elewant/AppBundle/Repository/HerdRepository.php

```diff
@@ -87,7 +88,7 @@ public function findOneByShepherdId(ShepherdId $shepherdId) :? Herd
         return $query->getOneOrNullResult();
     }
 
-    public function findOneByName(string $name) :? Herd
+    public function findOneByName(string $name):? Herd
```

Comment:
This one is still not conform the rules; it should be colon directly following the function arguments, and optional return value directly followed by return type